scripts/test.sh

```diff
@@ -22,7 +22,7 @@ if [ -n "$CIRCLE_BRANCH" ]; then
   if [[ $CIRCLE_BRANCH != $BASE_BRANCH ]] ; then
 
     #Run php lint, but only against modified php files
-    git diff --name-only origin/${BASE_BRANCH}... -- '*.php' | xargs -n1 php -l
+    git diff --name-only --diff-filter=M origin/${BASE_BRANCH}... -- '*.php' | xargs -n1 php -l
```

Comment:
I think it might be better to change to `--diff-filter=d` so it will ignore deletions, but still check all other things that have changed. - https://git-scm.com/docs/git-diff#git-diff---diff-filterACDMRTUXB82308203 In particular using `M` means we will miss files that are Added.